### DISCUSSION TITLE
Feature Request: Manage DNS records from the web frontend

### DISCUSSION THREAD

**Discussion by fr3fou** [vercel/vercel#4376] [General] [answered]
*Note: this is a cross post from spectrum*

This is a post regarding this tweet of mine:

https://twitter.com/fr3fou/status/1159760628216737793

In which I was stating my experience with managing DNS records on a new PC. I had to go through ALL of this:

- Installing Node
- Installing NPM
- Installing now

Logging in which took 30 mins (email was slow),
only for pointing a subdomain to my new project. 

All of this could've been easily skipped, had there been the option to manage the records from the web frontend.

If this gets implemented, I would consider moving all of my domains and projects to now, as this would make the entire platform "complete".

I've wanted this feature for ages now, I made my post on spectrum back in August, has there been any progress made since?

**Comment by mcsdevv** [comment, marked as answer]
Hello @fr3fou, I'm sorry for the time it took to achieve this. It's great timing on your post, we are looking to release this either this week or next.

Thanks again for bringing this up, this is something a lot of users have been after and I'm really pleased it's about to arrive. Once it's here, I'll post again to let you know!

**Reply by fr3fou** [reply]
Any news? Been a month already :/ I can't wait to use this

**Reply by mcsdevv** [reply]
Hey @fr3fou, this is now available for adding and deleting records, we're waiting to have edit support before we announce it but you can definitely use it! cc @impzero

**Comment by lyubchev** [comment]
What is up with that? Any news on final release date?

**Comment by lyubchev** [comment]
What is up with that? Any news on final release date?

**Comment by lyubchev** [comment]
Any news on final release date?

**Comment by mcsdevv** [comment]
Hello all, the DNS UI has now been released! You can read more about this here: https://vercel.com/blog/dns-records-ui
